### PR TITLE
Use Continuous Delivery pipeline to deploy v1.2.0-rc.0 to Rococo

### PR DESCRIPTION
**Pull request by MaciejBaj**
## Summary
Use Github Actions Pipeline to deploy v1.2.0-rc.0 to Rococo

## Checklist
- [ ] insert any notable changes, in order of commit

## Please check that your PR completes the requirements as follows
- [ ] Tests have been added for bug fixes/features
- [ ] Acceptance criteria from the issue has been completed

### CODE REVIEW

**Review by chiefbiiko — CHANGES_REQUESTED** [review]
We should omit the chain specs and genesis WASM because that must never change.

File: runtime/t0rn-parachain/src/lib.rs [hunk 1]
```diff
@@ -80,7 +80,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_version: 4,
     impl_version: 2,
     apis: RUNTIME_API_VERSIONS,
-    transaction_version: 2,
+    transaction_version: 3,
```
[the remark this answers is not in the document]
Comment (MaciejBaj) [reply runtime/t0rn-parachain/src/lib.rs:83]: only transaction_version needs bump from v1.1.0-rc.0

File: .github/workflows/circuit-t0rn-collator-release.yml [hunk 2]
```diff
@@ -173,11 +173,12 @@ jobs:
...
-#          curl -XPOST /
-#            -u ${{ secrets.GH_PAT }} /
-#            -H "Accept:application/vnd.github" /
-#            -H "Content-Type:application/json" /
-#            https://api.github.com/repos/t3rn/chainops/cd.yml/dispatches /
-#            --data "{\"action\":\"xtrigger\",\"tag\":\"${{ env.PUSHED_TAG }}\"}"
+  deploy-to-rococo:
+    runs-on: self-hosted
+    needs: release-on-github
+    steps:
```
Comment (chiefbiiko) [inline .github/workflows/circuit-t0rn-collator-release.yml:179]:
```suggestion
    steps:
      - name: 🏷️ Get the version tag
        run: echo "PUSHED_TAG=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
```

File: .github/workflows/circuit-t0rn-collator-release.yml [hunk 3]
```diff
@@ -173,11 +173,14 @@ jobs:
...
+      - name: 🏷️ Get the version tag
+        run: echo "PUSHED_TAG=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+      - name: 🚀 Shoot (Upgrade Runtime) latest release to Rococo
+        run: |
+          UPGRADE_AT_HEIGHT=$(($(curl -sSfH "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "chain_getHeader", "params": [ ] }' https://dev.net.t3rn.io/http-rpc | jq -r .result.number) + 0x64))
+          echo "UPGRADE RUNTIME AT:" ${{env.PUSHED_TAG}} ${UPGRADE_AT_HEIGHT} ${{env.PARACHAIN_NAME}}
+          ./scripts/upgrade-runtime-unsafe.sh ${{secrets.RUNTIME_UPGRADE_SEED}} wss://dev.net.t3rn.io https://dev.net.t3rn.io/http-rpc ${{env.PUSHED_TAG}} ${UPGRADE_AT_HEIGHT} ${{env.PARACHAIN_NAME}}
```
Comment (AwesomeIbex) [inline .github/workflows/circuit-t0rn-collator-release.yml:186]: This needs password too for ensuresignedby to work, it goes like this `${{secrets.RUNTIME_UPGRADE_SEED}}///${{secrets. RUNTIME_UPGRADE_SEED_PASS}}`

**Pull request merged**